docs/pages/components/switch.md

```diff
@@ -1,67 +1,24 @@
 ---
-title: Toggle / Switch
-id: toggle
-keywords: toggle, swtich
```

Comment:
maybe we can keep the keyword `toggle` for developers looking for that component. It will redirect them to `switch`?

Reply:
done